etc/baryssh-client.conf

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
```

Comment:
You probably won't be executing this script, nor will it have +x; shebang is not needed.

Other than that - server and client configuration is pretty much completely symmetrical and, as you will in normal cases run only one side on a single machine, I'd keep only one configuration file.